src/Nuget/Sarif.Sdk.nuspec

```diff
@@ -4,11 +4,11 @@
     <id>$id$</id>
     <version>$major$.$minor$.$patch$$prerelease$</version>
     <title>.NET Static Analysis Results Interchange Format (SARIF) Sdk</title>
-    <authors>Billy O'Neal,Larry Golding,Michael C. Fanning,Nick Guerrera</authors>
+    <authors>Billy O'Neal, Larry Golding, Michael C. Fanning, Nick Guerrera</authors>
```

Comment:
Also, neither Bill or Nick regard themselves as SDK owners, so no need to add them